shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx

```diff
@@ -67,33 +74,16 @@ To implement volume control and screen sharing in your <Vpl k="CLIENT" />, take
         agoraEngine.muteRemoteAudioStream(remoteUID, isMuted);
     }
     ```
-3.  **Configure <Vg k="ENGINE" /> to publish the screen sharing stream**
-
-    You use `ChannelMediaOptions` and the `updateChannelMediaOptions` method to specify the type of stream to publish. To switch between publishing screen and camera streams, in `preload.js`, add the following method before `window.onload = () => `:
-
-    ``` javascript
-    function updateChannelPublishOptions(publishScreenTrack)
-    {
-        // Update channel media options to switch between publishing screen and camera streams.
-        let channelOptions = new ChannelMediaOptions();
-        channelOptions.publishMicrophoneTrack = publishScreenTrack;
-        channelOptions.publishCameraTrack = !publishScreenTrack;
-        channelOptions.publishScreenTrack = publishScreenTrack
-        agoraEngine.updateChannelMediaOptions(channelOptions);
-    }
-    ```
 
-4.  **Display the screen sharing stream locally**
+3.  **Display the screen sharing stream locally**
```

Comment:
Would it be possible to use either stream or track everywhere?

Reply:
That's a good point, resolved